Rendering/OpenGL2/vtkOpenGLRenderer.cxx

```diff
@@ -997,3 +1069,116 @@ int vtkOpenGLRenderer::GetPickedIds(unsigned int atMost,
   }
   return k;
 }
+
+}
```

Comment:
this seems odd, are you sure you are not missing another commit ?

Reply:
Damn I had correct this automerge on the old PR